Resources/Styles/package.scss

```diff
@@ -35,6 +35,11 @@
           background-repeat: no-repeat;
           background-position: top 2px left;
         }
+        
+        li.history {
```

Comment:
I think this might be a copy/paste erropr. I think it should be `archived`.

Reply:
Ugh. Yes. Thanks for catching that!